Project-Descriptions-and-Plans/CVA6/CVA6-main-plan-approved-2024+.md

```diff
@@ -0,0 +1,524 @@
+- Mike Thompson, Director of Engineering, Verification Task Group
+- Florian Zaruba, Director of Engineering, HW & SW Task Groups
+- Davide Schiavone, Director of Engineering, Cores Task Group
+- Duncan Bees, Director, Technical Programs
+
+Florian's experience as the creator of ARIANE is also required for technical guidance.
+
+### Engineering resource supplied by members - requirement and availability
+
+The workplan below has been prepared according to available members' resources.
+
+### Marketing resource - requirement and availability
+
+The project needs support from:
+
+- Florian 'Flo' Wohlrab, OpenHW CEO
+- Michelle Clancy, Director of Marketing
+
+to promote CVA6 and attract new participants to the project.
+
+On the members' side, promotion activities (presentations, demos...) will mainly be addressed by the engineering team.
+
+### Funding for project aspects - requirement and availability
+
+Some marketing activities (OpenHW TV production...) might need OpenHW funding.
+
+Participating members provide the necessary tools to their teams.
+
+Refer to the acknowledgements at the bottom of this document for Horizon Europe and Chips JU funding.
+
+## Architecture and/or context diagrams
+
+![CVA6 architecture](https://github.com/openhwgroup/cva6/blob/master/docs/03_cva6_design/_static/ariane_overview.drawio.png)
+
+## Who would make use of OpenHW output
+
+Any entity needing a mid-range verified open-source RISC-V application processing core for ASIC and FPGA technologies:
+- OpenHW members
+- Large and small businesses
+- Academy and research
+- Future OpenHW projects
+
+Also refer to the "Market differentiators" section above.
+
+An open-source project favors collaborative and research projects that can start quickly without commercial and legal burdens.
+
+## Project license model
+
+The project artefacts and outputs will be licensed under Apache 2.0 or for SW code and Solderpad 2.0 or 2.1 for HW/RTL codes.
+
+Third-party open-source contributions will generally retain their own licence model.
+
+Linux and Yocto activities will retain the "upstream" licenses.
+
+"Viral" licences, such as GPL, will be avoided.
+
+## Description of initial code contribution, if required
+
+See [2023-06-26 PA document](https://github.com/openhwgroup/programs/blob/master/Project-Descriptions-and-Plans/CVA6/CVA6-plan-approved-2023-2024.md).
+
+## Repository Structure
+
+https://github.com/openhwgroup/cva6: the core master directory<br>
+/core: the CVA6 core (defined as the scope of the IP in the specification and targetting 100% coverage)<br>
+/corev_apu: the computing subsystem comprising CVA6<br>
+/docs: CVA6 documents (specification, users' guide...)<br>
+/verif: CVA6 verification, based on https://github.com/openhwgroup/core-v-verif
+	
+https://github.com/openhwgroup/core-v-verif: OpenHW verification framework
+	
+https://github.com/openhwgroup/cva6-sdk: RISC-V tools and BuiltRoot Linux
+
```

Comment:
We need to discuss with SW TG. The term "CORE-V-SDK" was adopted for CV32E40P but this doesn't really make sense. We need a nomenclature and definition of SDK across cores that has some consistency. @PaoloS02 should get involved.

Reply:
We have an [open position](https://careers.thalesgroup.com/global/en/job/R0233289/Ing%C3%A9nieur-de-recherche-en-logiciel-embarqu%C3%A9-pour-processeurs-RISC-V-H-F) to replace the contributor to this topic. As of now, we have no resource to progress on this remark.